Doc/whatsnew/3.15.rst

```diff
@@ -121,6 +121,13 @@ Deprecated
 Removed
 =======
 
+ctypes
+------
+
+* Removed the undocumented function :func:`!ctypes.SetPointerType`.
```

Comment:
```suggestion
* Removed the undocumented function :func:`!ctypes.SetPointerType`,
  which has been deprecated since 3.13.
```
(and similar in the NEWS blurb)